eisbuk-admin/src/components/atoms/AttendanceCard/AttendanceCard.tsx

```diff
@@ -36,28 +55,65 @@ const AttendanceCard: React.FC<Props> = ({
   const endTime = luxonEnd.toISOTime().substring(0, 5);
 
   const timeString = `${startTime} - ${endTime}`;
+
   return (
-    <div>
-      <div data-testid="time-string">{timeString}</div>
-      <div>{categories}</div>
-      {userBookings.map((user) => {
-        const isAbsent = absentees?.includes(user.customer_id) || false;
-        return (
-          <div key={user.customer_id}>
-            <div>{user.name}</div>
-            <button
-              type="button"
-              onClick={() =>
-                dispatch(markAttendance(user.customer_id, isAbsent))
+    <Container maxWidth="sm">
+      <List className={classes.root}>
+        <div key={id + "-" + timeString} className={classes.slotWrapper}>
+          <ListItem className={classes.listHeader}>
+            <ListItemText
+              primary={
+                <span>
+                  {timeString} <b>({userBookings.length})</b>
+                </span>
               }
-            >
-              {isAbsent ? "👎" : "👍"}
-            </button>
-          </div>
-        );
-      })}
-    </div>
+              secondary={translateAndJoinTags(categories, type)}
+            />
+          </ListItem>
+          {userBookings.map((user) => {
+            const isAbsent = absentees?.includes(user.customer_id);
+
+            return (
+              <UserAttendance
+                key={`${id}-${user.customer_id}`}
+                isAbsent={isAbsent!}
+                userBooking={user}
+              ></UserAttendance>
```

Comment:
UserAttendance wont receive any children so this should be written as `<UserAttendance {...props} />` rather than `<UserAttendance><UserAttendance />`
In more general sense, you can always use this with `jsx`, even with basic tags, even though HTML doesn't support this sometimes, i.e.
```jsx
<div />

// will get compiled to HTML as

<div></div>
```